cmd/rpc-common.go

```diff
@@ -0,0 +1,107 @@
+/*
+ * Minio Cloud Storage, (C) 2016 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"time"
+
+	"github.com/minio/dsync"
+)
+
+// Allow any RPC call request time should be no more/less than 3 seconds.
+// 3 seconds is chosen arbitrarily.
+const rpcSkewTimeAllowed = 3 * time.Second
```

Comment:
We can keep higher value for MS Windows only. However its better to fix the bug now.